docs/python/notebooks/onnxruntime-nuphar-tutorial.ipynb

```diff
@@ -0,0 +1,681 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Copyright (c) Microsoft Corporation. All rights reserved.  \n",
+    "Licensed under the MIT License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# ONNX Runtime: Tutorial for Nuphar execution provider\n",
+    "**Accelerating model inference via compiler, using Docker Images for ONNX Runtime with Nuphar**\n",
+    "\n",
+    "This example shows how to accelerate model inference using Nuphar.\n",
+    "\n",
+    "#### Tutorial Roadmap:\n",
+    "1. Create and run inference on a simple ONNX model, and understand how ***compilation*** works in Nuphar.\n",
+    "2. Create and run inference on a ***LSTM*** model, run symbolic shape inference, edit LSTM ops to Scan, and check Nuphar speedup.\n",
```

Comment:
typo: 'a LSTM' -> 'an LSTM'

Reply:
L is not vowel?

Reply:
For an acronym, it's based on the pronunciation of the first letter. In this case, it's "eL", where 'e' is  a vowel.